tools/NuGet/template-nuget/DynamoVisualProgramming.Tests.nuspec

```diff
@@ -26,16 +26,13 @@
     </metadata>
     <files>
         <!--These assemblies are referenced from the bin directory since they are not harvested for the installer-->
-       
         <file src="..\..\..\bin\AnyCPU\Release\DynamoCoreTests.dll" target="lib\net48" />
         <file src="..\..\..\bin\AnyCPU\Release\SystemTestServices.dll" target="lib\net48" />
         <file src="..\..\..\bin\AnyCPU\Release\TestServices.dll" target="lib\net48" />
-        <!--
+        <!--NET60 binaries, the path may change after we stop building NET48 solution-->
```

Comment:
theres also a duplicate of this file in template-artifactory that I think could also be modified so Revit can consume the net6 binaries.

Reply:
ah, I see this PR is only for public nugets.

Reply:
Thanks, will do in another PR